config/message.go

```diff
@@ -0,0 +1,52 @@
+// format for messages : ZFSOrder and ZFSResponse
+
+package config
+
+import (
+	"io/ioutil"
+)
+
+type ZFSOrder struct {
+	OrderUUID	 string	// mandatory
+	Action		 string // take_snapshot, send_snapshot, destroy_snapshot
+	Dataset		 string // tank/plop
+	SnapshotName	 string // Snapshot is dataset@snapshotname
+	SnapshotUUID	 string // mandatory
+	Destination	 string // hostname or ip for send
+	DestDataset	 string // dataset for recieve
+	SkipIfNotWritten bool   // should I take a snapshot if nothing is written
+	RollbackIfNeeded bool   // should I rollback if written is true on destination
+	SkipIfRenamed	 bool   // should I do the stuff if a snapshot has been renamed
+}
+
+// Status for response
+const (
+	WAS_RENAMED   = 1
+	WAS_WRITTEN   = 2
+	NOTHING_TO_DO = 3
+	ZFS_ERROR     = 4
```

Comment:
__[golint]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase

<!-- Comment by [SideCI](https://sideci.com) -->